spac-flow-docker-snapshot/action.yml

```diff
@@ -0,0 +1,78 @@
+name: SPAC flow docker snapshot
+description: SPAC flow docker snapshot
+
+inputs:
+  current_branch:
+    required: true
+  docker_labels:
+    required: false
+    default: |
+      maintainer=https://pagopa.it
+      org.opencontainers.image.source=https://github.com/${{ github.repository }}
+  enable_azure_devops_step:
+    required: false
+    default: false
+  azure_devops_apps:
+    required: false
+  azure_devops_project_url:
+    required: false
+  azure_devops_pipeline_name:
+    required: false
+  azure_devops_pat:
+    required: false
+    default: ''
+
+runs:
+  using: "composite"
+  steps:
+      - name: Log in to the Github Container registry
+        # https://github.com/docker/login-action/releases/tag/v3.3.0
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Set Docker tags
+        id: set_docker_tags
+        shell: bash
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "DOCKER_TAGS=ghcr.io/${{ github.repository }}:develop-latest,ghcr.io/${{ github.repository }}:snapshot" >> $GITHUB_OUTPUT
+          else
+            echo "DOCKER_TAGS=ghcr.io/${{ github.repository }}:snapshot,ghcr.io/${{ github.repository }}:snapshot-${{ inputs.current_branch }}" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Build and push Docker image
+        # https://github.com/docker/build-push-action/releases/tag/v6.9.0
+        uses: docker/build-push-action@4f58ea79222b3b9dc2c8bbdd6debcef730109a75
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.set_docker_tags.outputs.DOCKER_TAGS }}
+          labels: ${{ inputs.docker_labels }}
+
+      - name: Check azure_devops_pat
+        if: ${{ inputs.enable_azure_devops_step == 'true' }}
+        shell: bash
+        run: |
+          if [ -z "${{ inputs.azure_devops_pat }}" ]; then
+            echo "Error: azure_devops_pat is empty. This is required for triggering the Azure DevOps pipeline."
+            exit 1
+          fi
+
+          echo "🔨 Start launch trigger with Azure Devops"
+
+      - name: Trigger Azure DevOps pipeline
+        if: ${{ inputs.enable_azure_devops_step == 'true' }}
+        uses: jacopocarlini/azure-pipelines@v1.3
```

Comment:
@diegolagospagopa can we move this repo to pagopa organization?

Reply:
We did, we can only update all the ci/cd to be able to use.

This is only temporaly